src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml.cs

```diff
@@ -99,34 +160,72 @@ private void RawKeyDown(RawKeyEventArgs e)
                 return;
             }
 
-            const RawInputModifiers modifiers = RawInputModifiers.Control | RawInputModifiers.Shift;
-
-            if (e.Modifiers == modifiers)
+            switch (e.Modifiers)
             {
-#pragma warning disable CS0618 // Type or member is obsolete
-                var point = (Root as IInputRoot)?.MouseDevice?.GetPosition(Root) ?? default;
-#pragma warning restore CS0618 // Type or member is obsolete                
+                case RawInputModifiers.Control | RawInputModifiers.Shift:
+                {
+                    IControl? control = null;
+
+                    foreach (var popupRoot in GetPopupRoots(Root))
+                    {
+                        control = GetHoveredControl(popupRoot);
+
+                        if (control != null)
+                        {
+                            break;
+                        }
+                    }
+
+                    control ??= GetHoveredControl(Root);
 
-                var control = Root.GetVisualsAt(point, x =>
+                    if (control != null)
                     {
-                        if (x is AdornerLayer || !x.IsVisible) return false;
-                        if (!(x is IInputElement ie)) return true;
-                        return ie.IsHitTestVisible;
-                    })
-                    .FirstOrDefault();
+                        vm.SelectControl(control);
+                    }
+
+                    break;
+                }
 
-                if (control != null)
+                case RawInputModifiers.Control | RawInputModifiers.Alt when e.Key == Key.F:
```

Comment:
This isn't very discoverable (I had to look in the code for how to do it). Might it make sense to add this to the Options menu?

Reply:
Could do, but opening Options would already close the popup (since it loses focus)

Reply:
Yeah, but the main reason would be to give the user a clue that this option exists.